pkg/asset/machines/gcp/gcpmachines.go

```diff
@@ -69,8 +70,19 @@ func GenerateBootstrapMachines(name string, installConfig *installconfig.Install
 		Object: bootstrapGCPMachine,
 	})
 
+	// TODO: signedUrl
+	_, err := gcpinfra.ProvisionBootstrapStorage(installConfig, infraID)
+	if err != nil {
+		return nil, fmt.Errorf("failed to provision bootstrap resources: %w", err)
+	}
+
 	dataSecret := fmt.Sprintf("%s-%s", infraID, "bootstrap")
 	bootstrapCapiMachine := createCAPIMachine(bootstrapGCPMachine.Name, dataSecret, infraID)
+	bootstrapCapiMachine.Spec.Bootstrap.DataSecretName = ptr.To("user-data")
+	bootstrapCapiMachine.Spec.Bootstrap.ConfigRef = &v1.ObjectReference{
+		Name: "user-data",
+		// TODO: Add the signed url to json here.
+	}
```

Comment:
The ignition stub should be passed in the gcp capi provider in the `Ignition` hook--not here.